frontend/src/routes/moods/thoughts/+page.server.js

```diff
@@ -19,12 +19,42 @@ export const load = async ({ serverLoadEvent, cookies }) => {
 
     const thoughts = response.data.data;
     const thoughtsDate = response.data.meta.date;
+    const records = [];
+    const moods = [];
```

Comment:
I don't think this is used, if I'm right, then it would be nice to be removed and the call to the gateway below as well.

Reply:
I have tried it but infortunately if I remove them the for loop for gatherting the data doesn't work and I get 500 error